src/Altinn.Notifications.Persistence/Altinn.Notifications.Persistence.csproj

```diff
@@ -3,7 +3,6 @@
   <PropertyGroup>
     <TargetFramework>net7.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
-    <Nullable>enable</Nullable>
```

Comment:
We should try to keep this. If there are references we want to be nullable then mark them as such with a questionmark.